### PR TITLE
Kafkareadme: Updated the Kafka Destination readme file with the current syntax

### PR DESCRIPTION
**Pull request by vivinperis**
Updated the changes in syntax such as replacement of 'properties' with 'config', 'payload' with 'message'.
The docker-kafka implementation isn't working right now. Hence, on consultation with @furiel, I've replaced it with installation of kafka from official website which can be done following simple steps.

### CODE REVIEW

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build?
(admin: you have the next options (make sure you checked the code):
"ok to test" to accept this pull request (and further changes) for testing
"test this please" for a one time test run
"add to whitelist" add author of a Pull Request to whitelist (globally, be careful, it means this user can trigger kira for any PR)
do nothing -> CI won't start)

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build?
(admin: you have the next options (make sure you checked the code):
"ok to test" to accept this pull request (and further changes) for testing
"test this please" for a one time test run
"add to whitelist" add author of a Pull Request to whitelist (globally, be careful, it means this user can trigger kira for any PR)
do nothing -> CI won't start)

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build?
(admin: you have the next options (make sure you checked the code):
"ok to test" to accept this pull request (and further changes) for testing
"test this please" for a one time test run
"add to whitelist" add author of a Pull Request to whitelist (globally, be careful, it means this user can trigger kira for any PR)
do nothing -> CI won't start)

**Comment by Kokan** [comment]
@kira-syslogng ok to test

File: modules/kafka/README.md [hunk 1]
```diff
@@ -43,19 +43,46 @@ installed, compile with `--with-librdkafka`.
...
+[Kafka](https://kafka.apache.org/quickstart) can get you started in a minute:
 
+
+[Download](https://www.apache.org/dyn/closer.cgi?path=/kafka/2.5.0/kafka_2.12-2.5.0.tgz) the 2.5.0 release and un-tar it
+```
+tar -xzf kafka_2.12-2.5.0.tgz
+cd kafka_2.12-2.5.0
+```
+
+Start the ZooKeeper server once inside kafka_2.12-2.5.0 directory
```
Comment (Kokan) [inline modules/kafka/README.md:56]: 1. imho it is better not to replicate the content of the kafka quickstart here, link should be enough.
2. please keep kafkacat reference, that is a nice tool to use (you can just keep that oneliner about how to use it, everybody should be able to obtain it)

just out of curiosity, the `spotify/kafka` docker image are gone, or just does not build? (I would not build it myself if it is possible to just pull from docker.io)

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: modules/kafka/README.md (hunk 1 above)
Comment (vivinperis) [reply modules/kafka/README.md:56]: @konkan there are issues with building spotify/kafka directly and also with building it from docker image. It is unable to fetch some files. 
On the other hand, I found the official kafka guide to be much more simpler and easy to follow

Comment (furiel) [reply modules/kafka/README.md:56]: I agree with @Kokan . Let's just use a reference link. There are fix versions, download links that we should followup, and port back the changes when the quickstart guide changes.

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**